Remote Habits/AppDelegate.swift

```diff
@@ -25,6 +25,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         // Step 3: Register for push notifications
         UIApplication.shared.registerForRemoteNotifications()
+        
+        CustomerIO.config {
+            $0.logLevel = .debug
+        }
```

Comment:
1. I think that it would be more helpful if this code was inserted after `CustomerIO.initialize()` so that customers viewing this sample code would see an example of how to customize the SDK. 

```suggestion
  // Step 1: Initialise CIO SDK
  CustomerIO.initialize(siteId: workspaceId, apiKey: apiKey, region: Region.US)
  // Optionally, configure the SDK if you need to. 
   CustomerIO.config {
      $0.logLevel = .debug
   }
   // 2......
```

2. RH used to have auto screen view tracking enabled. Want to add that back to the configuration?

Reply:
Here you go ✅